src/routes/GetQuoteComponent.js

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line
+import GetQuote from '../components/GetQuote';
```

Comment:
- [ ] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks 👍🏿 


```suggestion
import GetQuote from '../components/getQuote';
```

- [ ] Please update your import to the correct path,  there is an error in the browser the first letter in getQuote is not capitalized 👍🏿

Reply:
Hi @prolajumokeoni ,

Thank you for the review. However, the path was referring to "GetQuote"- Capitalized as it should be. Your suggestion says that it's not capitalized but it is and the inline suggestions suggest that it should be in lowercase.  The code as is even if removing the disable eslint line works completely in the VSCODE no errors are being recorded. However, Pushing it to the repo it has a eslint error that suggests adding.js file extension to the referred to file path.  I have reinstalled the modules etc you can see in the commit history, but no such error occurs in my VScode or browser. See the snapshot attached 
<img width="1275" alt="image" src="https://github.com/George7h/math-magician/assets/101390796/5f6dd4d7-9062-4990-aaed-97255d0fac60"> And by adding the Github (online) eslint suggestion the whole site doenst work
 Any suggestions?

Reply:
Hi @prolajumokeoni ,

I have made the changes. For some reason in my VScode, the file names etc were correct, however, on Github the file names were not capitalized and didn't bring in the file name change. Thank you for steering me in the right direction.

Reply:
Hi @George7h, all the best with your next review, good job working on the requested changes 👍🏿